shawmut/bin/auto_offd.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env
+
+import time
+from datetime import datetime
+from subprocess import call
+import os.path
+import urllib2
+import json
+
+SCHEDULER_INTERVAL = 5
+ADDRESSES = [ '10.0.1.16', '192.168.13.6' ]
+
+# TODO capture timestamp instead of away_flag
+# TODO only turn on lights on night time
+
+def log(msg):
+    print "%s: %s" % (datetime.now().strftime('%Y-%m-%d %H:%M:%S'), msg)
+
+def check_if_home():
+    for ip in ADDRESSES:
+        DEVNULL = open(os.devnull, 'w') # TODO move outside loop
+        ip_status = call(['ping', ip, '-q', '-w', '1', '-c', '1'], stdout=DEVNULL, close_fds=True)
+        log("Found status of %s pinging %s" %(ip_status, ip))
+        if ip_status == 0:
+            return True
+    return False
+
+def have_guests():
+    os.path.isfile('/etc/shawmut/have_guests')
+
+def light_data():
+    data = urllib2.urlopen('http://localhost:5000/api/environment').read()
+    return json.loads(data)
```

Comment:
Might want some error handling here -- maybe even just wrap the whole thing in a try/except and log something/return {} if it fails?

Since we won't have any monitoring on this service at first, we need to be careful about crashes.